Editor/EditorCore/VertexManipulationTool.cs

```diff
@@ -43,6 +43,8 @@ internal static bool showHandleSettingsInScene
             get { return s_ShowHandleSettingsInScene; }
         }
 
+        public override bool gridSnapEnabled => Tools.pivotRotation == PivotRotation.Global;
+
```

Comment:
Should that default value be changed in trunk ?

Reply:
Yeah, I was a bit surprised why it's false by default, maybe @karl- would know the reasons behind that better? It seems that the default should be `Tools.pivotRotation == PivotRotation.Global;` instead but maybe that doesn't align well with the future work on snapping.

Reply:
I was the one adding this originally and the reason I think was that most tools might not need to use grid snapping so disable is might make sense. But now I see this, I wonder if that was the good solution.